sherpa/utils/__init__.py

```diff
@@ -232,16 +235,11 @@ def calc_ftest(dof1, stat1, dof2, stat2):
 
     Examples
     --------
+    >>> calc_ftest(11, 16.3, 10, 10.2) 
```

Comment:
@anetasie - given that it's easy to mess this up, should we include some text here to say what is being compared and what the result means. So, something like the original text but actually correct?

Reply:
help(calc_ftest) shows the information about the function and what it means. We may not need the doc update in this PR.
or you just referring to the test.

Reply:
I guess you want to update the descriptions in the Example section.

Reply:
Yes, I was talking about the examples. This PR removes text explaining the first example. Now, this is good because that text was wrong and the first example has changed, but I think it may be helpful to provide some explanation for the examples.

Reply:
Yes, we will do this in a separate PR